packages/web-twig/src/Resources/components/Accordion/AccordionContent.twig

```diff
@@ -0,0 +1,37 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _id = props.id -%}
+{%- set _labelledById = props.labelledById -%}
+{%- set _parent = props.parent | default(null) -%}
+{%- set _isOpen = props.isOpen | default(false) | boolprop -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Collapse' -%}
+{%- set _collapseClassName = _spiritClassPrefix ~ 'Collapse__content' -%}
+{%- set _accordionClassName = _spiritClassPrefix ~ 'Accordion__content' -%}
+{%- set _isOpenClassName = _isOpen ? 'is-open' : null -%}
+
+{# Attributes #}
+{%- set _idAttr = _id ? 'id="' ~ _id ~ '"' : null -%}
+{%- set _labelledIdAttr = _labelledById ? 'aria-labelledby="' ~ _labelledById ~ '"' : null -%}
+{%- set _parentAttr = _parent ? 'data-parent="' ~ _parent ~ '"' : null -%}
+
+{# Miscellaneous #}
+{%- set _collapseClassNames = [ _rootClassName, _isOpenClassName ] -%}
+{%- set _contentClassNames = [ _accordionClassName, _class ] -%}
+{%- set _mainPropsWithoutId = props | filter((value, prop) => prop != 'id') -%}
```

Comment:
A question: don't we have to also filter out the `aria-labelledby` and `data-parent` attributes from here?

Reply:
We probably don't have to filter it at all here. But I think it is inconsistent in all components, so I will link into it in a separate PR.

Reply:
> it is definitely out of the scope of this issue

Sure it is. It just popped onto my mind when I was looking at how we handle the `id` but not the other attributes.

> I will look into it in a separate PR

Oh, that's more than we could wish 🙂. Thank you!